apps/backoffice-v2/src/main.tsx

```diff
@@ -15,6 +15,22 @@ import './i18n';
 import './index.css';
 import dayjs from 'dayjs';
 import advancedFormat from 'dayjs/plugin/advancedFormat';
+import posthog from 'posthog-js';
+
+if (
+  !window.location.host.includes('127.0.0.1') &&
+  !window.location.host.includes('localhost') &&
+  env.VITE_REACT_APP_PUBLIC_POSTHOG_KEY &&
+  env.VITE_REACT_APP_PUBLIC_POSTHOG_HOST
+) {
+  posthog.init(env.VITE_REACT_APP_PUBLIC_POSTHOG_KEY!, {
+    api_host: env.VITE_REACT_APP_PUBLIC_POSTHOG_HOST!,
```

Comment:
Considering the `if` check the `!` (bangs) shouldn't be needed, is this not the case?